distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/Select2,Apache Software License 2.0.txt

```diff
@@ -0,0 +1 @@
+Select2,GNU General Public License v2.0 only.txt
```

Comment:
The way I would recommend reading the license data is to start with the xml or the html. Then, if additional information is desired for a particular component's license, either use the remote URL or access the local file. The regular filenames make it easy to programmatically find local files when processing the xml data. The symlinks avoid the bloat this regular naming scheme would otherwise cause.